woodwork/column_accessor.py

```diff
@@ -165,15 +164,20 @@ def semantic_tags(self):
     def __eq__(self, other):
         if self._schema != other._schema:
             return False
+        if self._series.name != other._series.name:
+            return False
         if isinstance(self._series, pd.Series):
             return self._series.equals(other._series)
         return True
 
     def __getattr__(self, attr):
         # If the method is present on the Accessor, uses that method.
+        # If the method is present on TableSchema, uses that method.
         # If the method is present on Series, uses that method.
         if self._schema is None:
             _raise_init_error()
+        if hasattr(self._schema, attr):
+            return self._make_schema_call(attr)
```

Comment:
We didn't previously have a need for this, but unless we make `use_standard_tags` a property on the `WoodworkColumnAccessor`, we will still need a `_make_schema_call` method.

Reply:
there's a code coverage issue below because we don't have any callable attrs... I'd rather not update the `make_schema_call` to not have the callable case, so I'm now thinking it's better to remove this and add a property on the column accessor instead... @thehomebrewnerd

Reply:
Added `use_standard_tags` as a property on the Column Accessor